src/DatePicker/stories/index.stories.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import { Story, ComponentMeta } from '@storybook/react'
+import { Form, FormItemProps } from 'antd'
+import { differenceInDays } from 'date-fns'
+import { default as DatePicker } from '../index'
+
+export default {
+  title: 'DatePicker',
+  component: DatePicker
+} as ComponentMeta<typeof DatePicker>
+
+type Args = {
+  formItem: FormItemProps
+  datePicker: typeof DatePicker
+}
+
+const Template: Story<Args> = (args: Args) => {
+  return (
+    <Form.Item style={{ width: '320px' }} {...args.formItem}>
+      <DatePicker
+        style={{ width: '320px' }}
+        {...args.datePicker}
+        disabledDate={current => {
+          return current && differenceInDays(current, new Date()) < 0
+        }}
+      />
+    </Form.Item>
+  )
+}
+
+export const Basic = Template.bind({})
+
+Basic.args = {
+  datePicker: {
+    label: 'DatePicker',
+    showToday: false,
+    format: 'DD/MM/YYYY'
+  },
+  formItem: {
+    label: 'Date'
+  }
+}
+
+export const BasicQuarter = Template.bind({})
+
+BasicQuarter.args = {
+  datePicker: {
+    label: 'DatePicker',
+    picker: 'quarter'
```

Comment:
Create a different template for the quarter picker and use the `QuarterPicker` exported from index.ts